apps/deploy-web/src/utils/deploymentData/v1beta3.ts

```diff
@@ -22,6 +25,51 @@ const getDenomFromSdl = (groups: any[]): string => {
   return denoms[0];
 };
 
+export function appendTrialAttribute(yamlStr: string) {
+  const sdl = getSdl(yamlStr, "beta3", networkStore.selectedNetworkId);
+
+  Object.keys(sdl.data.profiles.placement).forEach(placement => {
+    if (!sdl.data.profiles.placement[placement].attributes) {
```

Comment:
`sdl.data.profiles.placement[placement]` is used like 10+ times. would be easier to read if it's aliased by a var